source/web-backends.rst

```diff
@@ -60,6 +60,29 @@ To set the default backend to an application listening on port 1024 (for example
 
   [isabell@philae ~]$ 
 
+available options
+-----------------
+
++-----------------+-------------+---------------------------------------------+ 
+| parameter       | value       | description                                 | 
++=================+=============+=============================================+ 
+| --apache        | N/A         | sets the backend to apache                  |
+|                 |             | (Default - use for htaccess/Wordpress/etc.) |
++-----------------+-------------+---------------------------------------------+ 
+| --static        | N/A         | Use the nginx static file server            |
```

Comment:
There is no `--static` at the moment. It was part of our original design document, but hasn't made it into the code (so far). Where did you get that from? :grin:

Reply:
[https://manual-72.uberspace.de/background-http-stack.html#nginx](https://manual-72.uberspace.de/background-http-stack.html#nginx) in the last sentence it's mentioned.